.buildkite/pipelines/on_merge.yml

```diff
@@ -186,6 +186,14 @@ steps:
         - exit_status: '-1'
           limit: 3
 
+  - command: .buildkite/scripts/steps/archive_so_migration_snapshot.sh target/plugin_so_types_snapshot.json
+    label: 'Extract Saved Object migration plugin types'
+    depends_on: build
```

Comment:
maybe it doesn't need to depend on build, it could run in parallel, because it will build it's own dev-kibana either way

Reply:
Ack, I'll remove it then.
```suggestion
```